servers/rendering/renderer_rd/shaders/particles_copy.glsl

```diff
@@ -159,7 +159,10 @@ void main() {
 			} break; //nothing
 			case TRANSFORM_ALIGN_Z_BILLBOARD: {
 				mat3 local = mat3(normalize(cross(params.align_up, params.sort_direction)), params.align_up, params.sort_direction);
-				local = local * mat3(txform);
+				mat3 angle = mat3(vec3(cos(particles.data[particle].custom.x), -sin(particles.data[particle].custom.x), 0.0),
```

Comment:
Won't this break user shaders that use `z-billboard` but also use ``CUSTOM`` for their own custom data?

Also, this needs to match the Compatibility renderer if we are keeping this change https://github.com/godotengine/godot/blob/master/drivers/gles3/shaders/particles_copy.glsl

Reply:
Yes, it will, but that case they shouldn't use the billboard.

Reply:
Ping on this. This is going to break unexpectedly for users that rely on custom.x. Can we get away without relying on custom.x here? Maybe with a different shader variant?

Reply:
I don't think there's people using the built-in billboarding and using custom.x as something else, but i can remove it